### DISCUSSION TITLE
Preset Missing

### DISCUSSION THREAD

**Discussion by AshGreh** [danny-avila/LibreChat#4686] [Troubleshooting] [answered]
### What happened?

The option to add presets is missing on 0.7.5 when using Anthropic and AWS Bedrock.

### Steps to Reproduce

1. Build 0.7.5 vs 0.7.4 and run locally, practically default settings on the .env with the exception of the region being changed

### What browsers are you seeing the problem on?

Chrome

### Screenshots

![image](https://github.com/user-attachments/assets/852b838d-a393-4783-9081-31794ed15525)
![image](https://github.com/user-attachments/assets/08b86192-2421-4e67-927c-cb706bdb8f31)
![image](https://github.com/user-attachments/assets/0716bb58-c1e2-437b-a175-3bcca5089d70)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
It exists in the right-hand side panel now

**Reply by AshGreh** [reply]
I'm stupid, thank you for the help!

**Reply by saeedesmaili** [reply]
@danny-avila  This guide should probably be updated then?
https://www.librechat.ai/docs/user_guides/presets

I tried making a PR, but I don't understand where in the repo should I add screenshots?
<img width="943" alt="image" src="https://github.com/user-attachments/assets/cc898584-0a26-4db5-be08-eb8fbbe54482" />

**Comment by alexlyzhov** [comment]
When I click presets, it says "No presets yet, use the settings button to create one". I looked at preset creation everywhere in settings and couldn't find it, I was very confused. Consider changing this message to mention that preset creation has moved to the right hand side panel.

**Reply by alexlyzhov** [reply]
@danny-avila

**Comment by Nihisil** [comment]
Were you able to find out how to create presets? The document is outdated, and I cannot find it anywhere in the right panel. It is frustrating

**Reply by alexlyzhov** [reply]
It should be at the bottom of the Parameters tab

**Reply by saeedesmaili** [reply]
<img width="297" alt="image" src="https://github.com/user-attachments/assets/ccebfab9-d284-48a7-8e45-486c562052ac" />

<img width="365" alt="image" src="https://github.com/user-attachments/assets/63a545f5-11e8-46be-8a28-a33cdfea9987" />

**Reply by Nihisil** [reply]
Thank you. Before sending a message, I was checking the demo at https://librechat-librechat.hf.space/c/new and it does not have parameters. Now it clear how it suppose to work

![image](https://github.com/user-attachments/assets/03743e10-e4e9-4c53-a1b1-804db9961980)

**Reply by danny-avila** [reply]
I recommend using Agents as a successor to presets, which also brings in more capabilities.

https://www.librechat.ai/docs/features/agents
